src/platforms/javascript/common/configuration/webworkers/index.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Web Workers
+description: "Learn how to use Sentry's Browser SDK in Web Workers API."
+keywords: ["webworkers"]
+sidebar_order: 9000
+notSupported:
```

Comment:
@rhcarvalho, I added the `supported` tag with `javascript` (for browser, I could not find a specific selector for browser), I'm not sure if that's necessary as this file is under javascript's common content.

Still had to keep `notSupported` unfortunately to prevent this doc from showing up in SDKs, as they all are falling back to javascript.

Reply:
Hmm yeah the `supported` filter implies all guides too 😢 .

@PeloWriter do you know of an existing solution for this particular case?

Can we show some content only for JavaScript Browser (and not Angular, Cordova, Ember, React, Vue, etc) without having to list all guides in the `notSupported` filter? My point was that the not supported list here should be "everything except browser" and it is hard to keep that up-to-date.

Reply:
Sadly, no workaround yet exists. If the content displays for the primary platform, the only way to turn it off for the associated frameworks is to individually list them as `notSupported`.

This would be another valuable use of the [release registry as a source of truth](https://github.com/getsentry/sentry-release-registry/issues/39) - it would prevent manual management that is hard to keep up to date.

Reply:
What if we don't put this under `common/`?

There must be a way, in the worst case we can add a new flag `includeInGuides: false` or similar.
Since this is not intended to be shared, putting it outside of common content would be an easier start.